esphome/components/debug/debug_component.h

```diff
@@ -1,18 +1,41 @@
 #pragma once
 
 #include "esphome/core/component.h"
+#include "esphome/core/helpers.h"
```

Comment:
which requires the `macros.h` header
```suggestion
#include "esphome/core/macros.h"
#include "esphome/core/helpers.h"
```